src/NativeScript/inspector/GlobalObjectConsoleClient.cpp

```diff
@@ -115,12 +115,20 @@ void GlobalObjectConsoleClient::takeHeapSnapshot(JSC::ExecState*, const String&
 }
 
 void GlobalObjectConsoleClient::time(JSC::ExecState*, const String& title) {
-    m_consoleAgent->startTiming(title);
+    std::unique_ptr<String> startMsg = m_consoleAgent->startTiming(title);
+    const String* url;
+    if (startMsg) {
+        ConsoleClient::printConsoleMessage(MessageSource::ConsoleAPI, MessageType::Timing, MessageLevel::Warning, *startMsg.get(), (const String&)url, -1, -1);
```

Comment:
According to the specification https://console.spec.whatwg.org/#time there should be no logging when `console.time()` is called, unless a timer with the same label has already been started.

Reply:
@Pip3r4o - that's the case here too. The `startMsg` will always be `nullptr` unless you have called `console.time()` twice with the same token. Then a warning will be printed.